### PR TITLE
Remove Bazel build

### PR DESCRIPTION
**Pull request by metcalf**
This removes the Bazel build generated by sbt-bazel in favor of just publishing packages for internal consumption. I confirmed that I can build Zoolander against this.

cc @mio-stripe

### CODE REVIEW

File: version.sbt [hunk 1]
```diff
@@ -1 +1 @@
-version in ThisBuild := "0.2.3-SNAPSHOT"
```
Comment (metcalf) [inline version.sbt:1]: Do we want to retain the `SNAPSHOT` suffix on the `develop` branch? Publish this as rc1 and then immediately add `-SNAPSHOT`?

Comment (avi-stripe) [reply version.sbt:1]: yes, I think we can publish `0.2.3-rcX` and then go back to `0.2.3-SNAPSHOT`.

File: build.sbt [hunk 2]
```diff
@@ -221,39 +188,7 @@ lazy val shadedAsm = project.
...
-        |jar_jar(
-        |    name = "asmTreeShaded",
-        |    input_jar = "@org_ow2_asm_asm_tree//jar",
-        |    rules = "shade_rule",
-        |    visibility = [
-        |      '//visibility:public',
-        |    ],
-        |)
-      """.stripMargin)
+    releaseVersion := { ver => ver }
```
Comment (avi-stripe) [inline build.sbt:191]: I don't know sbt well enough but could this be used to avoid us having to hand-modify the `version` all the time?

**Comment by avi-stripe** [comment]
I think it's ok to merge this as is and followup with the version numbering stuff

**Review by avi-stripe — APPROVED** [review]

**Comment by metcalf** [comment]
I think there's value in committing the actual RC version changes to the develop branch to make it easier to figure out what a particular artifact came from. I'd be inclined to update the version to `0.2.3-rc1-SNAPSHOT` and publishing this internally to that end.

**Pull request merged**